install_remove_pod/after/execution_output.txt

```diff
@@ -7,35 +7,16 @@ Inspecting targets to integrate
   Using `ARCHS` setting to build architectures of target `Pods-SampleApp`: (`$(ARCHS_STANDARD_64_BIT)`)
 
 Finding Podfile changes
-  R JSONKit
```

Comment:
I dont think this is right, given this integration spec is supposed to be testing removing a pod